test/e2e/antreapolicy_test.go

```diff
@@ -4300,6 +4336,7 @@ func TestAntreaPolicy(t *testing.T) {
 		t.Run("Case=ACNPPriorityConflictingRule", func(t *testing.T) { testACNPPriorityConflictingRule(t) })
 		t.Run("Case=ACNPRulePriority", func(t *testing.T) { testACNPRulePriority(t) })
 		t.Run("Case=ANPPortRange", func(t *testing.T) { testANPPortRange(t) })
+		t.Run("Case=ACNPSourcePort", func(t *testing.T) { testACNPSourcePort(t) })
```

Comment:
Maybe put it under L4325 `t.Run("Case=ACNPPortRange", func(t *testing.T) { testACNPPortRange(t) })`

Reply:
Done